### PR TITLE
Feature/top twenty words

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i refactor top_20_words.py file in practice-2, update function main and add module top_20_words to main.py
#48

### CODE REVIEW

**Comment by hothanhtuan1209** [comment]
image

File: python/practices/practice-3/src/path2/top_20_words.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+def main():
+    """
+    Read a text file, clean the words, and print the 20 most frequently
+    used words in the ebook.
+    """
+
+    print(os.path.basename(__file__))
+    ebook_text = read_file(EBOOK)
+    cleaned_words = clean_words(ebook_text)
+    word_counts = count_words(cleaned_words)
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/top_20_words.py]: Can we rename to: `total_number_of_words`?

File: python/practices/practice-3/src/path2/top_20_words.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    top_count = 20
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/top_20_words.py]: Please define the constant for this number

File: python/practices/practice-3/src/path2/top_20_words.py (hunk 1 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/top_20_words.py]: Done

File: python/practices/practice-3/src/path2/top_20_words.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/top_20_words.py]: Done

File: python/practices/practice-3/src/constants/constants.py [hunk 3]
```diff
@@ -8,3 +8,7 @@
 WORD = './src/texts/words.txt'
 MARKOV = './src/texts/markov.txt'
 EBOOK = './src/texts/ebook.txt'
+
+
+# Constant for counts top
+TOP_COUNT = 20
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/constants/constants.py:15]: What is the TOP_COUNT?

Comment (hothanhtuan1209) [reply python/practices/practice-3/src/constants/constants.py:15]: I have updated the comment for constant

**Pull request merged**